server/actions/InventoryItems.ts

```diff
@@ -127,6 +126,33 @@ const softDeleteRequestPipeline: PipelineStage[] = [
   },
 ]
 
+function searchAggregate(search: string): PipelineStage {
+  return {
+    $match: {
+      $or: [
+        { 'itemDefinition.name': { $regex: search, $options: 'i' } },
+        {
+          'itemDefinition.category.name': {
+            $regex: search,
+            $options: 'i',
+          },
+        },
+        { 'assignee.name': { $regex: search, $options: 'i' } },
+        { name: { $regex: search, $options: 'i' } },
+        { email: { $regex: search, $options: 'i' } },
```

Comment:
These fields don't exist, add filtering by attributes

Reply:
attributes added to new story